src/Templates/Views/elements/textbox.pode

```diff
@@ -38,8 +38,15 @@ $(
             }
 
             $value = [string]::Empty
+            $textareavalue = [string]::Empty
             if (![string]::IsNullOrWhiteSpace($data.Value)) {
                 $value = "value='$($data.Value)'"
+                $textareavalue = "$($data.Value)"
+            }
```

Comment:
To save initialising 2 variables with the same content, I'd check if the textbox type is multiline and set `$value` accordingly:
```powershelll
$value = [string]::Empty
if (![string]::IsNullOrWhitespace($data.Value)) {
    if ($data.Multiline) {
        $value = $data.Value
    }
    else {
        $value = "value='$($data.Value)'"
    }
}
```

So if the value happens to be 500,000 chars in length, we don't consume double the size in 2 vars 😄

Reply:
Hi, just pushed changes as per your comment.